$NetBSD: patch-ac,v 1.11.2.2 2010/11/23 21:14:45 spz Exp $

Back port of patch for CVE-2010-2252 for to version 1.12 of "wget" taken
from Debian:

http://packages.debian.org/sid/wget
http://ftp.de.debian.org/debian/pool/main/w/wget/wget_1.12-2.1.debian.tar.gz

--- src/init.c.orig	2009-09-22 04:02:41.000000000 +0100
+++ src/init.c	2010-11-22 18:53:25.000000000 +0000
@@ -243,6 +243,7 @@
   { "timeout",          NULL,                   cmd_spec_timeout },
   { "timestamping",     &opt.timestamping,      cmd_boolean },
   { "tries",            &opt.ntry,              cmd_number_inf },
+  { "trustservernames", &opt.trustservernames,  cmd_boolean },
   { "useproxy",         &opt.use_proxy,         cmd_boolean },
   { "user",             &opt.user,              cmd_string },
   { "useragent",        NULL,                   cmd_spec_useragent },
